app/assets/src/components/common/CoverageVizSidebar/CoverageVizSidebar.jsx

```diff
@@ -0,0 +1,367 @@
+      data.coverage,
+      data.coverage_bin_size
+    );
+
+    this.histogram = new Histogram(this.histogramContainer, [coverageData], {
+      labelY: "Coverage",
+      domain: [0, data.total_length],
+      skipBins: true,
+      showStatistics: false,
+      colors: ["#A9BDFC"],
+      barOpacity: 1,
+      margins: {
+        left: 170,
+        right: 40,
+        top: 30,
+        bottom: 30
+      },
+      numTicksY: 2,
+      labelYOffset: 15,
+      labelYLarge: true
+    });
+    this.histogram.update();
+  };
+
+  renderGenomeViz = data => {
+    const contigsReadsData = this.processContigsReadData(
+      data.contigs,
+      data.reads
+    );
+
+    this.contigsReadGenomeViz = new GenomeViz(
+      this.contigsReadsContainer,
+      contigsReadsData,
+      {
+        domain: [0, data.total_length]
+      }
+    );
+    this.contigsReadGenomeViz.update();
+  };
+
+  getAccessionOptions = () => {
+    return this.props.params.accessionSummaries.map(summary => ({
+      text: summary.id,
+      value: summary.id
+    }));
+  };
+
+  getAccessionMetrics = () => {
+    if (!this.state.currentAccessionData) {
+      return {};
+    }
+
+    const referenceAccession = (
+      <BasicPopup
+        trigger={
+          <a
+            href={`https://www.ncbi.nlm.nih.gov/nuccore/${
+              this.state.currentAccession.id
+            }?report=genbank`}
+            target="_blank"
+            rel="noopener noreferrer"
+            className={cs.ncbiLink}
+          >
+            {this.state.currentAccession.id}
+          </a>
+        }
+        inverted
+        content={this.state.currentAccession.name}
+        horizontalOffset={13}
+      />
+    );
+
+    return {
+      referenceAccession,
+      referenceLength: this.state.currentAccessionData.total_length,
+      alignedContigs: this.state.currentAccession.n_contig,
+      avgAlignedLength: "--",
+      avgCoverageDepth: "--",
+      coverageBreadth: "--",
+      alignedReads: this.state.currentAccession.n_read,
+      avgMismatchedPercent: "--"
+    };
+  };
+
+  renderContentBody = () => {
+    if (!this.state.currentAccessionData) {
+      return (
+        <div className={cs.loadingContainer}>
+          <div className={cs.loadingMessage}>
+            <LoadingIcon className={cs.loadingIcon} />Loading Visualization...
+          </div>
+        </div>
+      );
+    }
+
+    const metrics = this.getAccessionMetrics();
+
+    return (
+      <div className={cs.body}>
+        <div className={cs.stats}>
+          {METRIC_COLUMNS.map((col, index) => (
+            <div className={cs.column} key={index}>
+              {col.map(metric => (
+                <div className={cs.metric} key={metric.key}>
+                  <div className={cs.label}>
+                    <BasicPopup
+                      trigger={<div>{metric.name}</div>}
+                      inverted
+                      content={METRIC_TOOLTIPS[metric.key]}
```

Comment:
seems like the tooltips of `METRIC_TOOLTIPS` should be  inside of `METRIC_COLUMNS` because they are only used in the same context

Reply:
Sounds great, thanks.